### ISSUE TITLE
Gradient descent class

### ISSUE THREAD

**Issue by HippolyteGuigon** [HippolyteGuigon/Feed_Forward_Neural_Network_Implementation#12]
Code a Gradient Descent class that inherits from Neural Network attributes
